.github/workflows/prerelease.yml

```diff
@@ -0,0 +1,99 @@
+name: Pre-release tests
+
+on:
+  # Runs at 5:00 UTC on Mondays
+  schedule:
+    - cron: "0 5 * * 1"
+  workflow_dispatch:
+
+jobs:
+
+  make-check-job:
+    name: Run 'make check'
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: Install dependencies
+        run: pip install -r requirements-dev.txt
+      - name: Installing Avocado in develop mode
+        run: python3 setup.py develop --user
+      - name: Run make check
+        run: make check
```

Comment:
If `make check` is already executed every PR during our gating process, this makes me think about the real need of this.

I mean, if this is needed is because by some reason we accepted a PR with this failing. Yes, double-checks are good, but again, makes me think about this.

Reply:
@beraldoleal 
Good question. I asked myself the same while adding the job. I ended adding it because it's in the checklist https://github.com/avocado-framework/avocado/blob/master/examples/testplans/release/pre.json#L10
so I thought there might be some historical reason for this. I'm fine removing it in both places in the PR and there. Thoughts ?

Reply:
Unless we have a strong reason for keeping it, we probably could remove this. It will force us double checking before merge.

@clebergnu @willianrampazzo @richtja can you guys comment here, please?

Reply:
IMO we don't need this. As @beraldoleal said, is the same check form PR gating process.

Reply:
Looking at https://github.com/avocado-framework/avocado/actions, I don't see any checks running on the `master` branch. This means that 2 branches that succeeded in running `make check` may fail when both are merged to `master`, as one may eventually impact the other. In this case, I don't see a problem leaving the `make check` in the weekly test as it is testing master.

Reply:
@willianrampazzo `.gitlab/workflows/ci.yml` does have a `make check` already that is executed on every PR that is trying to be part of "master".  Not sure if I got your point.

Reply:
Imagine I send a PR to change the software manager utility. The `make check` run on my PR succeed and is ready to be merged. Imagine someone else sends a PR at the same time I sent mine changing the software manager runner on nrunner side changing it and do not account for the changes I submitted on my PR. This PR will also succeed in running `make check`. Both PRs will be good to go because the changes on both PRs will not conflict. With both PRs merged, an issue was introduced and makes `make check` fail, but we are not running `make check` on master and the issue will appear later when someone testing its own PR find a fail on `make check` related to it.

I don't think running `make check` weekly will totally solve this problem, but at least we may see the error before someone finds then by accident. Also, I don't see one more run of `make check` per week as a problem here.

Reply:
Thanks @willianrampazzo , of course! That is the strong reason that I was missing. :)

Reply:
@willianrampazzo I still do not understand to it. We are running the `make check` on master after every merge and that should catch this issue, am I right? https://github.com/avocado-framework/avocado/runs/3372112014

Reply:
@richtja yes, but we might have some kind of "race condition" when merging commits. There is a low possibility but the check doesn't consume so much resources.

Reply:
@richtja, my comment was because of the removal of the Travis jobs that ran on master. Indeed, in the link you mentioned, I see they are running on the master, but for non-x86 architectures. If that is enough, I don't see a problem removing the `make check` here.

Reply:
I don't have a problem to have `make check` here, I just wanted to mention that we are testing the master branch and IMO that is enough.